RenderSystems/GLSupport/src/EGL/X11/OgreX11EGLWindow.cpp

```diff
@@ -94,6 +94,10 @@ namespace Ogre {
             NameValuePairList::const_iterator opt;
             NameValuePairList::const_iterator end = miscParams->end();
 
+            if ((opt = miscParams->find("externalWlDisplay")) != end)
```

Comment:
you can do the same in OgreWaylandWindow for "externalWindowHandle" and "parentWindowHandle" as those indicate an X11 Window now

Reply:
Done